README.md

```diff
@@ -8,6 +8,12 @@
 [![Build Status](http://ci.decidim.codegram.com/api/badges/codegram/decidim/status.svg)](http://ci.decidim.codegram.com/codegram/decidim)
 [![codecov](https://codecov.io/gh/codegram/decidim/branch/master/graph/badge.svg)](https://codecov.io/gh/codegram/decidim)
 
+## Requirements
+
+* Ruby 2.3.1
+* PostgreSQL 9.5 or newer with the `hstore` extension
```

Comment:
Yes, but you have to explicitly create the extension, so it's not enabled by default. I guess I can remove this part.

Actually, the `hstore` module is installed since v9.0, but I'm not sure if we're using newer things or not :/